sympy/complexanalysis/Contour Integration.py

```diff
@@ -0,0 +1,39 @@
+from sympy import *
+import matplotlib.pyplot as plt
+
+class ComplexIntegral:
```

Comment:
Probably inheriting `Basic` would be better.

Reply:
These files are from another pull request for a different issue. I don't know why these are showing up here. This PR is only meant to be for by_parts_integration, not contour integration.

Reply:
Maybe you are working from the same branch on your system for both the PRs.

Reply:
This file is not in my cloned repo. I have made this on the desktop and added it to a new branch from there not through bash but github.com.

Reply:
Can't say, most probably, it is something related to branch issues.

Reply:
I think now it's gone.